stats/src/main/java/com/facebook/stats/mx/StatsReader.java

```diff
@@ -36,6 +36,8 @@
   public MultiWindowDistribution getDistribution(String key);
   public String getAttribute(StatType key);
   public String getAttribute(String key);
+  public Callable<Long> getDynamicCounter(StatType key);
+  public Callable<Long> getDynamicCounter(String key);
```

Comment:
what's the use case for wanting the callable versus just the value?  what's the difference between

  stats.getAttribute(key)
vs
  stats.getDynamicCounter(key).call()

when would you want to be responsible for call() directly versus just asking the stats object for the current value?

further, as the user of the stats object, you will be setting the callable...if you need it outside the context of the stats object, why not save it?

i guess i don't know the use case